notebooks/week1/01.dataExploraton.py

```
@@ -0,0 +1,145 @@
+def process_file(filename, dirname):
+    filepath = os.path.join(dirname, filename, 'part-0.parquet')
+    df = spark.read.parquet(filepath)
+    df = df.drop('step')
+    # Ensure 'id' column is included
+    if 'id' not in df.columns:
+        df = df.withColumn('id', df['relative_date_PCIAT'])  # Use an existing column or create a new one
+    return df.toPandas(), filename.split('=')[1]
+
+def load_time_series(dirname) -> pd.DataFrame:
+    # List all subdirectories in the specified path
+    directories = [file.path for file in dbutils.fs.ls(dirname) if file.path.endswith("/")]
```

Ruff

Undefined name `dbutils`

(F821)